app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -65,8 +76,9 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
         continue;
       }
 
-      if (pr.base!.ref != 'master') {
-        log.fine('This change is not staged to land on master, skipping.');
+      final String? baseRef = pr.base!.ref;
+      if (baseRef != 'master' && baseRef != 'main') {
```

Comment:
```suggestion
      if (baseRef != Config.defaultBranch(pr.base!.repo!.slug())) {
```

Reply:
Nice! I'm assuming you'll update the `defaultBranch` method when flutter/flutter switches to the `main` branch?

Reply:
Yep! Infra is maintaining this API for the default branch instead of hard coding it everywhere